google/cloud/security/iam/explain/filters.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" filter classes """
+
+from datetime import datetime
+# pylint: disable=missing-type-doc, missing-param-doc
+# pylint: disable=missing-return-type-doc, missing-return-doc
+# pylint: disable=singleton-comparison
+
+class TimeFilter(object):
+    """Time Filter class."""
+    def __init__(self, explaine_filter=None):
+        """To generate a python filter from protobuf filter"""
```

Comment:
ditto: please document the parameters

You can see examples here:
http://www.sphinx-doc.org/en/stable/ext/example_google.html